presto-main/src/main/java/com/facebook/presto/metadata/HandleResolver.java

```diff
@@ -53,6 +55,8 @@ public HandleResolver()
         handleResolvers.put("$system", new MaterializedHandleResolver(new SystemHandleResolver()));
         handleResolvers.put("$info_schema", new MaterializedHandleResolver(new InformationSchemaHandleResolver()));
         handleResolvers.put("$empty", new MaterializedHandleResolver(new EmptySplitHandleResolver()));
+
+        functionHandleResolvers.put("static", new MaterializedFunctionHandleResolver(new StaticFunctionNamespaceHandleResolver()));
```

Comment:
Do you wanna mangle this in case we have a connector called `static`?

Reply:
Though why would a connector call itself static lol